vector/src/main/java/im/vector/app/features/settings/VectorSettingsGeneralFragment.kt

```diff
@@ -322,6 +326,8 @@ class VectorSettingsGeneralFragment :
 
             false
         }
+        // Account deactivation is visible only if account is not managed by an external URL.
```

Comment:
This isn't quite right:

Account deactivation is visible only if the homeserver is not using MSC3861 delegated auth.

This is because the external account management URL field `account` is optional in [MSC2965](https://github.com/sandhose/matrix-doc/blob/msc/sandhose/oidc-discovery/proposals/2965-oidc-discovery.md#proposal)

Whether a homeserver is using MSC3861 can be determined by look at whether there is a `m.authentication` (or `org.matrix.msc2965.authentication` whilst in MSC process) entry in the `.well-known/matrix/client` entry.

I think a new a new boolean field on `HomeServerCapabilities` something like `delegatedOidcAuth` would make sense.

Reply:
Yes, that is correct. 👍

Reply:
OK, I will wait for #8620 to be merged, since I will need code that I have added there, to resume the work on this PR (after a rebase).